common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BrokerOperationBundle.java

```diff
@@ -60,6 +60,8 @@ public enum Operation {
         MSAL_HELLO(API.MSAL_HELLO, BrokerAccountManagerOperation.HELLO),
         MSAL_GET_INTENT_FOR_INTERACTIVE_REQUEST(API.ACQUIRE_TOKEN_INTERACTIVE, BrokerAccountManagerOperation.GET_INTENT_FOR_INTERACTIVE_REQUEST),
         MSAL_ACQUIRE_TOKEN_SILENT(API.ACQUIRE_TOKEN_SILENT, BrokerAccountManagerOperation.ACQUIRE_TOKEN_SILENT),
+        MSAL_FETCH_DCF_AUTH_RESULT(API.FETCH_DCF_AUTH_RESULT, BrokerAccountManagerOperation.FETCH_DCF_AUTH_RESULT),
+        MSAL_ACQUIRE_TOKEN_DCF(API.ACQUIRE_TOKEN_DCF, BrokerAccountManagerOperation.ACQUIRE_TOKEN_DCF),
```

Comment:
Enum constants should be moved to the end. Also, whyo we have account manager operation for this call? We are only going to do a content provider call right?

Reply:
ah good catch, fixed now. thanks!